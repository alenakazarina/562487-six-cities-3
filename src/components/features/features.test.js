import React from 'react';
import renderer from 'react-test-renderer';
import Features from './features';

const features = {
  type: `Apartment`,
  bedrooms: 3,
  maxAdults: 6
};

describe(`Features`, () => {
  it(`should render Features correctly`, () => {
    const tree = renderer.create(
        <Features features={features} />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
