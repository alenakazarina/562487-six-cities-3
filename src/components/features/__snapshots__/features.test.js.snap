// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Features should render Features correctly 1`] = `
<ul
  className="property__features"
>
  <li
    className="property__feature property__feature--entire"
  >
    Apartment
  </li>
  <li
    className="property__feature property__feature--bedrooms"
  >
    3
     Bedrooms
  </li>
  <li
    className="property__feature property__feature--adults"
  >
    Max 
    6
     adults
  </li>
</ul>
`;
